plugins/notion/src/notion.ts

```diff
@@ -371,15 +508,14 @@ async function processItem(
                 fieldId: field.id,
                 message: `Value is missing for field ${field.name}`,
             })
-            continue
```

Comment:
Removing this on purpose. A single field not having a value should not exclude it entirely from synchronizing.